api/app/src/Model/DeputyshipProcessingRawDbAccess.php

```diff
@@ -77,6 +77,15 @@ public function insertOrder(array $insertOrder): DeputyshipProcessingRawDbAccess
         try {
             $qb = $this->ingestWriterEm->getConnection()->createQueryBuilder();
 
+            $qb->insert('client')
```

Comment:
The test needs a client with ID 1 to be inserted so that it will pass, but the client **MUST NOT** be inserted here, only in the test.

The candidate should have a reference to an existing client already. We should not be adding the client when we insert the order, as this will cause chaos. Each insert/update function on this class should only insert/update a court_order, court_order_deputy, or court_order_report row. If it inserts anything else, it potentially interferes with or disrupts existing data.